libs/community/langchain_community/chat_models/hunyuan.py

```diff
@@ -124,114 +73,84 @@ class ChatHunyuan(BaseChatModel):
     For more information, see https://cloud.tencent.com/document/product/1729
     """
 
-    @property
-    def lc_secrets(self) -> Dict[str, str]:
-        return {
-            "hunyuan_app_id": "HUNYUAN_APP_ID",
-            "hunyuan_secret_id": "HUNYUAN_SECRET_ID",
-            "hunyuan_secret_key": "HUNYUAN_SECRET_KEY",
-        }
-
-    @property
-    def lc_serializable(self) -> bool:
-        return True
-
-    hunyuan_api_base: str = Field(default=DEFAULT_API_BASE)
-    """Hunyuan custom endpoints"""
-    hunyuan_app_id: Optional[int] = None
-    """Hunyuan App ID"""
-    hunyuan_secret_id: Optional[str] = None
-    """Hunyuan Secret ID"""
-    hunyuan_secret_key: Optional[SecretStr] = None
-    """Hunyuan Secret Key"""
```

Comment:
changing the names of init args is a breaking change — do we need to make it?

Reply:
Thank you for your reminder. 
In addition to remove the `hunyuan_api_base` and `hunyuan_app_id ` which maybe unused in newest hunyuan api,
maybe I should keep `hunyuan_secret_id` and `hunyuan_secret_key` instead of `tencentcloud_secret_id` and `tencentcloud_secret_key`.

Reply:
> Thank you for your reminder. In addition to remove the `hunyuan_api_base` and `hunyuan_app_id ` which maybe unused in newest hunyuan api, maybe I should keep `hunyuan_secret_id` and `hunyuan_secret_key` instead of `tencentcloud_secret_id` and `tencentcloud_secret_key`.

I have completed the changes described above and tested it again ^^